### ISSUE TITLE
[Bug]: show indexes：USER dump has no privilege to execute this stmt

### ISSUE THREAD

**Issue by goodMan-code** [matrixorigin/matrixone#6034]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

DDL : 
create database test;
use test;
drop table if exists t;
create table t(
	a int,
	b int,
	c int,
	primary key(a)
);

show indexes from t;
this stmt return value is : 
ERROR 20101 (HY000): internal error: do not have privilege to execute the statement

**Comment by goodMan-code** [comment]
p0

**Comment by daviszhen** [comment]
it was fixed at #6230

**Comment by goodMan-code** [comment]
I have Recheck it, can be closed. on commit commit 45c43c065b169c8dc00c9f8ed6964dac3e0a6ef8
